templates.dist/estate/default_detail.php

```diff
@@ -33,6 +33,17 @@
 	<?php
 	$pEstates->resetEstateIterator();
 	while ( $currentEstate = $pEstates->estateIterator() ) { ?>
+		<?php
+		if($currentEstate["onoffice-settings-title-and-description"])
```

Comment:
`<head>` is usually done by the WP theme.  I think this whole thing needs to run a lot sooner, similar to [EstateViewDocumentTitleBuilder](https://github.com/onOfficeGmbH/oo-wp-plugin/blob/057cca241c8efe32bea6514ddf3e46f764e1122e/plugin.php#L131-L133).